client/src/components/MultiJurisdictionAudit/CSVForm/index.tsx

```diff
@@ -109,6 +111,17 @@ const CSVFileForm = ({
                 </FormSectionDescription>
               </FormSection>
             )}
+            {filePurpose === 'cvrs' && (
+              <FormSection>
+                <H4>Cast Vote Records</H4>
+                <FormSectionDescription>
+                  Click &quot;Browse&quot; to choose the appropriate Cast Vote
+                  Records (CVR) file from your computer. This file should be a
+                  comma-separated list of all the ballots counted by your
+                  tabulator, in order.
+                </FormSectionDescription>
+              </FormSection>
```

Comment:
maybe just pull these out into a `title` and `description` prop to replace the `filePurpose` prop?